src/python/pants/backend/python/dependency_inference/scripts/general_dependency_visitor.py

```diff
@@ -91,6 +91,7 @@ def visit_ImportFrom(self, node):
         self._visit_import_stmt(node, abs_module + ".")
 
     def visit_TryExcept(self, node):
+        previous_weaken = self.weaken_strong_imports
```

Comment:
Can we just drop this file's change?

The Rust-based parser is going to be the one-and-only way of parsing deps. So if it is ahead feature-wise of the Python-based parser, the better.

Reply:
Good point, I'm fine dropping the python changes. Given that 2.18 has the cutover to the rust parser as the default, I could see it making more sense to leave the python one unaltered to keep it as a "legacy" toggle.